tests/chart_tests/test_git_sync_relay_deployment.py

```diff
@@ -181,3 +181,52 @@ def test_gsr_deployment_without_ssh_credentials_and_known_hosts(self, kube_versi
             "--root=/git",
         ]
         assert c_by_name["git-daemon"]["livenessProbe"]
+
+    def test_gsr_deployment_with_resource_overrides(self, kube_version):
+        """Test that gitsync  deployment are configurable with custom resource limits."""
+        resources = {
+            "requests": {"cpu": 0.1, "memory": "512Mi"},
+            "limits": {"cpu": 0.1, "memory": "512Mi"},
```

Comment:
We should set these to something obviously wrong. In other areas we've had to adjust default resources, and when test cases aren't obviously wrong, sometimes those future adjustments end up being identical to custom value test cases. Something like
```suggestion
            "requests": {"cpu": 99.5, "memory": "777Mi"},
            "limits": {"cpu": 99.6, "memory": "888Mi"},
```